proseco/catalog.py

```diff
@@ -199,6 +209,12 @@ def _get_aca_catalog(**kwargs):
     aca.guides = get_guide_catalog(stars=aca.acqs.stars, fids=aca.fids, mons=aca.mons,
                                    img_size=img_size_guide, **kwargs)
 
+    # Set output catalog aca.n_guide to reflect what the number of requested
+    # guide stars as determined in guide star selection processing. The input
+    # arg value of n_guide (which initially defines aca.n_guide) reflects the
+    # requested number of available slots for guide + monitor stars / windows.
+    aca.n_guide = aca.guides.n_guide
```

Comment:
OK.  So I've gotten confused again.  From https://github.com/sot/proseco/pull/346#issuecomment-797572645 I thought the n_guide attribute was already going to be set to the number of selected guide stars as a side effect of the processing, but maybe I didn't completely validate.

Reply:
> Also, it may help that with splitting out monitors to a new mons table, the attribute n_guide will correctly reflect the number of guide stars selected. It just won't necessarily match the input n_guide arg

This PR is making that so. It *was* true previously for the the `aca.guides.n_guide` attribute, but not for `aca.n_guide`.

Reply:
Thanks.  That's helpful for context.  So now from the comment I'm just confused about if aca.guides.n_guide is set to the number of guide stars requested of the guide algorithm or the number of guide stars selected by the guide algorithm.  I'll review.

Reply:
OK.  I see for example that the sparkles code sets a local n_guide equal to the number of guide stars in the selected catalog and for its warning compares that to the attribute n_guide with, IIRC, the idea that is the helpful number of guide stars requested of the guide algorithm.  I think "number of requested guide stars as determined in guide star selection processing" is still a bit confusing but is a step in the right direction and we can come back to this if we come up with better language.